tables/automl/predict_test.py

```diff
@@ -43,15 +43,15 @@ def test_predict(capsys):
         "MaritalStatus": "divorced",
         "Month": "jul",
         "PDays": 4,
-        "POutcome": '0',
+        "POutcome": "0",
         "Previous": 12,
     }
 
     ensure_model_online()
-    automl_tables_predict.predict(PROJECT, REGION, STATIC_MODEL, inputs)
+    automl_tables_predict.predict(PROJECT, REGION, STATIC_MODEL, inputs, True)
     out, _ = capsys.readouterr()
-    assert 'Predicted class name:' in out
-    assert 'Predicted class score:' in out
+    assert "Predicted class name:" in out
+    assert "Predicted class score:" in out
```

Comment:
should we add an assertion for the feature importance too?

Reply:
sure. although these tests don't do much.

Reply:
Ty - it's more to be setting a good example and displaying good practices imho

Reply:
important note: feature importance might not be in _every_ predict call